.github/workflows/build.yml

```diff
@@ -74,11 +74,11 @@ jobs:
       matrix:
         operating-system: [ubuntu-latest, macos-latest, windows-latest]
         php: ['8.0', '8.1', '8.2']
+        # We need to use the ~ operator to prevent weirdness on Windows.
+        # See https://github.com/composer/composer/issues/10943
         dependencies:
-          - 'symfony/validator:^4.4'
-          - 'symfony/validator:^5'
-          - 'guzzlehttp/guzzle:^6.5'
-          - 'guzzlehttp/guzzle:^7.2'
+          - 'symfony/validator:~4.4'
+          - 'symfony/validator:~5.0'
```

Comment:
since the commit message for this was "specify 5.0 with tilde to prevent updating to symfony 6" but drupal 10 I think requires symfony 6 would this prevent php-tuf from being used on drupal 19?

Reply:
The other thing that prevents Symfony 6 (at the moment) is the requirement on `symfony/validator:^4.4 || ^5` in composer.json. That needs to be fixed in a follow-up PR, because you're right: until we allow Symfony 6, PHP-TUF is incompatible with Drupal 10.

Reply:
I am don't know enough about the interdepedencies of `symfony/*` packages but this jump out at me